src/App.tsx

```diff
@@ -12,6 +13,7 @@ class App extends React.Component {
         <div>
           <Route path='/' exact={true} component={Lobby} />
           <Route path='/game/:id' component={Game} />
```

Comment:
Base routes and specifik routes should be devided. Base routes in the outer structure and for the game or lobby should be structured in their folders.